### DISCUSSION TITLE
Ubuntu Summit 2023 was a success - Alan Pope's blog

### DISCUSSION THREAD

**Discussion by giscus** [popey/popey.com-blog#44] [General]
# Ubuntu Summit 2023 was a success - Alan Pope's blog

Last week, I wrote about my somewhat last-minute plans to attend the 2023 Ubuntu Summit in Riga, Latvia. The event is now over, and I’m back home collating my thoughts about the weekend.
The tl;dr: It was a great, well-organised and run event with interesting speakers.
Here’s my “trip report”.
Logistics The event was held at the Radisson Blu Latvija. Many of the Canonical staff stayed at the Raddison, while most (perhaps all) of the non-Canonical attendees were a short walk away at the Tallink Hotel.

https://popey.com/blog/2023/11/ubuntu-summit-2023-was-a-success/

**Comment by arubislander** [comment]
Great write-up. Thanks for sharing! Wish I could have been there myself. But if you stopped by the UBports stand you might have noticed my PinePhone being displayed :-)
(btw, the foundation, community and project is called UBports. The OS is (still) called Ubuntu Touch)
